src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -299,6 +299,54 @@ public virtual async Task PatchOne_Update_Test()
                 );
         }
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a computed field is missing from the request body.
+        /// </summary>
+        [TestMethod]
+        public virtual async Task PatchOneWithComputedFieldMissingFromRequestBody()
+        {
+            // Validate successful execution of a PATCH update when a computed field (here 'last_sold_on_date')
+            // is missing from the request body. In such a case we don't try to put NULL value for such a field while performing
+            // the update.
+            string requestBody = @"
+            {
+                ""book_name"": ""New book"",
+                ""copies_sold"": 50
+            }";
+            string expectedLocationHeader = $"id/1";
+
+            await SetupAndRunRestApiTest(
+                    primaryKeyRoute: expectedLocationHeader,
+                    queryString: null,
+                    entityNameOrPath: _entityWithReadOnlyFields,
+                    sqlQuery: GetQuery("PatchOneUpdateWithComputedFieldMissingFromRequestBody"),
+                    operationType: EntityActionOperation.UpsertIncremental,
+                    requestBody: requestBody,
+                    expectedStatusCode: HttpStatusCode.OK
+                );
+
+            // Validate successful execution of a PATCH insert when a computed field (here 'last_sold_on_date')
+            // is missing from the request body. In such a case we don't try to put NULL value for such a field while performing
```

Comment:
_In such a case we don't try to put NULL value for such a field while performing_ 

The sql query doesn't validate the state of the `last_sold_on_date` column in this test. A request resulting in a query like:
```sql
 UPDATE books_sold
  SET last_sold_on = null where id = 1
```
both the `last_sold_on` and `last_sold_on_date` would be updated to null.

Reply:
I updated the sql query to validate the state of last_sold_on_date and last_sold_on columns as well.

Reply:
Added those for MsSql/PgSql. For MySql, I think there is a bug as to how the default values are parsed. Opened an issue here: https://github.com/Azure/data-api-builder/issues/1696.